src/site/content/en/blog/idle-detection/index.md

```diff
@@ -52,33 +45,13 @@ can limit these calculations to moments when the user interacts with their devic
 | 1. Create explainer                      | [Complete][explainer]    |
 | 2. Create initial draft of specification | [Complete][specification]|
 | 3. Gather feedback & iterate on design   | [In progress](#feedback) |
-| 4. Origin trial                          | [In progress][ot]        |
-| 5. Launch                                | Not started              |
+| 4. Origin trial                          | Completed                |
+| 5. **Launch**                            | **Completed**            |
```

Comment:
Is this what we always do? Seems like we should say version it landed in.

Reply:
Now we'd use the new widget ([example](https://web.dev/file-system-access/#browser-support)), but the compatibility data for Idle Detection is not yet on MDN.